Code/Framework/AzFramework/AzFramework/Physics/Common/PhysicsJoint.h

```diff
@@ -136,4 +136,19 @@ namespace AzPhysics
             AZStd::vector<AZ::Vector3>& lineBufferOut,
             AZStd::vector<bool>& lineValidityBufferOut) = 0;
     };
+
+    //! Interface to access editor-only Joint utilities and helper functions
+    class EditorJointHelpersInterface
+    {
+    public:
+        AZ_RTTI(AzPhysics::EditorJointHelpersInterface, "{79B0CE51-E7DA-4CA9-BAE0-8441E09B4713}");
+
+        EditorJointHelpersInterface() = default;
+        virtual ~EditorJointHelpersInterface() = default;
+        AZ_DISABLE_COPY_MOVE(EditorJointHelpersInterface);
+
+        virtual AZStd::unique_ptr<AzPhysics::JointConfiguration> ComputeOptimalJointLimit(
+            const AzPhysics::JointConfiguration* currentConfiguration,
+            const AZStd::vector<AZ::Quaternion>& exampleLocalRotations) = 0;
```

Comment:
What's wrong with the existing name?

Reply:
Maybe it's just me, but if I read `localRotationSamples` is clear that "this is a list of rotation samples". But `exampleLocalRotations` I read it like "list of rotation taken from an example..." what is the example? is there an example asset? (these are  question maybe users could ask themselves).

It was just a suggestion. You can take it or leave it.

Reply:
There is another function which uses the same terminology, although that parameter never got used so it could be cleaned up. I'd like to change that in a separate PR, which is the reason I was hesitant to change the name here, but maybe it does make sense. I'll also add a doxygen comment to explain what the purpose of the parameter is.